test/test_csv.py

```diff
@@ -48,6 +49,24 @@ def test_sym(self):
     def test_the(self):
         print("List of config parameters : {}".format(the))
         return True
+    
+    def test_data(self):
+        data = Data(r'..\data\auto93.csv')
```

Comment:
I think we should use the['file'] here. I know it's the same in the lua code. But, the --file arg is not being used anywhere.